spec/unit/provider/mount/mount_spec.rb

```diff
@@ -247,15 +247,6 @@
       expect(@provider.current_resource.enabled).to be_falsey
     end
 
-    it "should set enabled to false if the mount point is not last in fstab" do
```

Comment:
It seems like we still probably want this test. I'm confused why we're removing it.

Reply:
I removed the [code](https://github.com/chef/chef/pull/10472/files#diff-ee080140ef5e117fe81d326426a69ea8b1ef913dee4ed1267fe07350f7b59610L73-L76) because this was causing multiple entry in `fstab`, that's why I need to remove this test.